tipocket-ctl/tpctl/deploy.py

```diff
@@ -14,28 +14,51 @@ def __init__(self, env, feature, case, binary, image, params):
         self.binary = binary
         self.image = image
         self.params = params
+        self._is_cron = params['cron'] is True
```

Comment:
I think we can construct deploy_id here.

Reply:
Nailed.